### ISSUE TITLE
tf.keras.utils.plot_model(bert_classifier, show_shapes=True, dpi=48) throws and Error

### ISSUE THREAD

**Issue by owos** [tensorflow/models#10821]
# Prerequisites

Please answer the following questions for yourself before submitting an issue.

- [x] I checked to make sure that this issue has not been filed already.
- [x] I am using the latest TensorFlow Model Garden release and TensorFlow 2.
- [x] I am reporting the issue to the correct repository. (Model Garden official or research directory)

## 1. The entire URL of the file you are using

https://github.com/tensorflow/models/blob/master/docs/nlp/fine_tune_bert.ipynb

## 2. Describe the bug

In cell  34, of the notebook, the line that plots the classifier model model throws up an error

## 3. Steps to reproduce
To reproduce, just pick the notebook and run all cells
Steps to reproduce the behavior.

## 4. Expected behavior

It is supposed to plot t he model after running it.

## 5. Additional context

`InvocationException error:  Program terminated with status: 1. stderr follows: Error: bad label format `

## 6. System information

Colab, 4/11/2022

**Comment by laxmareddyp** [comment]
Hi @owos,

I have replicated the code [here](https://colab.sandbox.google.com/gist/laxmareddyp/9aee3effd67279914b32514f23160fe9/fine_tune_bert.ipynb) using tensor-flow version 2.9.2 and it's working fine , can you please check it. we are working on this issue internally and will update on the same.

**Comment by owos** [comment]
Great!, it works now!
